src/pages/Profile.tsx

```diff
@@ -97,13 +97,15 @@ type ProfileContent = {
   username: string;
   followerCount: number;
   followingCount: number;
+  followStatus: boolean;
```

Comment:
Another thing I noticed is that `followStatus` doesn't get updated right away when you click the follow/unfollow button. It only updates after a new fetch happens, so for a better experience we need to mutate the state in the front end as soon as the user clicks the button